src/app/server/Server.cpp

```diff
@@ -560,6 +580,9 @@ void Server::Shutdown()
 
     chip::Dnssd::Resolver::Instance().Shutdown();
     chip::app::InteractionModelEngine::GetInstance()->Shutdown();
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+    chip::app::InteractionModelEngine::GetInstance()->SetICDManager(nullptr);
```

Comment:
```suggestion
    app::InteractionModelEngine::GetInstance()->SetICDManager(nullptr);
```
And yes, I know the stuff right above this has the extra prefixes.  It shouldn't.